packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -78,12 +76,22 @@ const SupportListItem = ({ item }: { item: SupportItem }) => {
   );
 };
 
-export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
-  const { items } = useSupportConfig();
+type SupportButtonProps = {
+  title?: ReactChild;
+  supportItems?: SupportItem[];
+  displayMode?: 'customOnly' | 'all' | '';
```

Comment:
The `''` mode shouldn't be needed. In my mind this is the same as not provided at all since it's optional and would be `undefined` in that case.

Reply:
I also realize the name `customOnly` was my suggestion, but the name doesn't feel like it's clear enough imo. Thoughts on something else? Another option is we could just use a boolean flag if we think that only 2 cases are needed.